src/services/email.service.ts

```diff
@@ -35,28 +33,10 @@ export class EmailService {
     };
   }
 
-  static buildSendGridTransport(): Transporter<SMTPTransport.SentMessageInfo> {
-    const credentials = {
-      api_user: SENDGRID_API_USER as string,
-      api_key: SENDGRID_API_KEY as string
-    };
-    const sendGrigOptions = sgTransport(this.getSendGridOptions(credentials));
-    return nodemailer.createTransport(sgTransport(sendGrigOptions));
-  }
-
-  static buildSesTransport(): Transporter<SESTransport.SentMessageInfo> {
-    const ses = SESService.createSES();
-    return nodemailer.createTransport({
-      SES: { ses, aws }
-    });
-  }
-
-  static async sendEmail(email: EmailInterface.IEmail, emailPlatform: string) {
+  static async sendEmail(email: EmailInterface.IEmail) {
```

Comment:
I'm a little bit afraid of this interface `EmailInterface.IEmail` because it has props that are not clear like the `context` one, and the `template`.
What do you think? Am I crazy?

Reply:
Yep, I agree ! It's fixed now ! Thanks !